src/nanoemoji/glue_together.py

```diff
@@ -0,0 +1,37 @@
+from fontTools.ttLib.tables import otTables as ot
+from fontTools import ttLib
+import os
+
+_ORIGINAL="/tmp/Bungee7.ttf"
+_COLR="/tmp/bungee7/AnEmojiFamily.ttf"
+
+font = ttLib.TTFont(_ORIGINAL)
+colr = ttLib.TTFont(_COLR)
+_OUTPUT = '/tmp/bungee7/Bungee_SVG_COLR.ttf'
+
+# Copy all glyphs used by COLR over
+_glyphs_to_copy = set()
+
+def _collect_glyphs(paint):
+  if paint.Format == ot.PaintFormat.PaintGlyph:
+    _glyphs_to_copy.add(paint.Glyph)
+
+for record in colr["COLR"].table.BaseGlyphList.BaseGlyphPaintRecord:
+  record.Paint.traverse(colr["COLR"].table, _collect_glyphs)
+
+for glyph_name in _glyphs_to_copy:
+  print("Copy glyph", glyph_name)
+  font['glyf'].glyphs[glyph_name] = colr['glyf'].glyphs[glyph_name]
```

Comment:
I would put a check to make sure we are not overwriting any pre-existing glyphs that happen to have the same name. Actually, you can define the glyph name in the destination otsvg font to be something unique, it need not be the same as the glyph name in the source colr font.

Reply:
While the glyph names in the source and destination font need not be the same, it helps if they are, because you can then simply copy the COLR table as is from one font to the other and not bother about the glyph names in the PaintGlyph or PaintColrGlyph structures.
If you change the destination names (which you may have to do in case they clash with existing glyph in destination font) you must also make sure to rename any occurrences of those glyphs in the COLR table or else it either fails to compile (if can't find the referenced glyph with old name) or leads to incorrect results as it may reference the wrong glyph.

Reply:
the fontTools.merge tool is a good reference for how to merge tables from multiple fonts (the latter might eventually gain support for merging fonts with COLR table, but this particular task at hand requires a more spacial handling than the general merge tool).
If you look at https://github.com/fonttools/fonttools/blob/ac4d5611ac793e4815f64eb05d46e0cb46ec4612/Lib/fontTools/merge/__init__.py#L73-L88, you'll see it first computes a "mega" glyph order where it handles duplicate glyph names by appending an incremental `#<digit>`; then it reloads all the fonts from disk again (to ensure that no table is loaded) and immediatly does `setGlyphOrder` using the possibly modified glyphOrder (plus additional `renameCFFCharStrings` because CFF is alas always special), before loading the rest of the tables; this way, e.g. when you decompile COLR, you'll get the renamed glyph names (that are guaranteed not to clash in the destination font).

Reply:
For context, intent is to take a much simpler path than a "real" merge here. We build the two fonts to have matching glyph names and then trust that assumption. Since we are adding COLR+glyf to an SVG I think the glyf entries used by COLR can safely take priority?

Reply:
ok, but at least fail if in the unlikely circumstances when that assumption does not hold

Reply:
Taking Bungee as an example, I don't want to keep the fallback glyfs from the original SVG font so I'm ok with overwriting them.

Reply:
why you don't want to keep the fallback glyfs? That's not a decision that a tool like this should make on behalf of the font developer. Overwriting any existing glyfs that are not related to COLR effectively changes the functionality of the original font and defeats the purpose of the tool which should be "add COLR while keep the rest exactly as it was".
Since the original font can have any sorts of glyph names, and the way we name PaintGlyph layer glyphs is using an simple format like `{color_glyph.glyph_name}.{i}` (see `_next_name` in `write_font.py), it's entirely possible that a clash may occur: e.g. the font developer assigns an alternate form of glyph "a" for whatever reason (say some GSUB substitution) and names it "a.1", etc.
We need to prevent overwriting from happening, and ideally handle them by renaming the clashing glyphs so both existing and new glyphs are kept (we can do that in a separate PR if you prefer).

Reply:
> why you don't want to keep the fallback glyfs?

The intent is that a max color font has color for all major browsers so nobody should ever see the fallback. Selfishly viewing myself as the main user keeping fallbacks just makes me do more work. That said, I buy the notion this should be at least optional, maybe a flag, on by default?

Reply:
are you deliberately dropping the glyf fallback glyphs? or are you just silently ignoring the fact that some of them may be overwritten when copying over the glyf layers used by COLR table?
We could have an option to make them all blank if we deem them unnecessary for web use, but if a font developer took the pain to draw them for them to be displayed as fallback in environments that can only display b/w, it would be a pity to break those.
I noticed that a lot of color fonts out there have b/w fallback glyphs, especially the text typefaces, non-emoji ones. The fact that nanoemoji doesn't know/care about making those doesn't mean they can be indiscriminately chopped away.

Reply:
It's a fari point. How about destroy non color glyphs is an option, on by default, and in that case I'll actually scrub glyf entirely?

(I might not implement the retention option initially)

Reply:
sgtm

Reply:
Added `--destroy_non_color_glyphs`, default true, if false errors out